packages/jest-circus/src/formatNodeAssertErrors.ts

```diff
@@ -76,6 +76,9 @@ const getOperatorName = (operator: string | undefined, stack: string) => {
   if (stack.match('.throws')) {
     return 'throws';
   }
+  if (stack.match('.fail')) {
+    return 'fail';
+  }
```

Comment:
This `stack.match` fallback is only needed for Node versions older than v10.11 because it wan't until v10.11.0 that the `AssertionError` of `assert.fail` had an `operator` value set by default (https://github.com/nodejs/node/pull/22694).

Reply:
Maybe mention that in a comment, it's common to grep the codebase for things like `node 10` when phasing out support for an EOL version